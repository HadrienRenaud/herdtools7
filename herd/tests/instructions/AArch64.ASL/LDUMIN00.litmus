AArch64 LDUMIN00
{
x = 0;
0:X1 = -128;
0:X3 = x;
}

P0                 ;
LDUMIN W1, W2, [X3] ;

forall 0:X2=0 /\ [x]=-128


