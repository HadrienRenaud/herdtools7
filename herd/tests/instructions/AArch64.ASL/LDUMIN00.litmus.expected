Test LDUMIN00 Required
States 1
0:X2=0; [x]=0;
No
Witnesses
Positive: 0 Negative: 1
Condition forall (0:X2=0 /\ [x]=-128)
Observation LDUMIN00 Never 0 1
Hash=3adec7fbf6ac5bf7d7cc1898364d85ec

